tests/generic_relations_regress/models.py

```diff
@@ -218,3 +218,14 @@ def prevent_deletes(sender, instance, **kwargs):
 
 
 models.signals.pre_delete.connect(prevent_deletes, sender=Node)
+
+
+class EventTarget(models.Model):
```

Comment:
I might group these with `Place`/`Restaurant`/etc, so that all the _inheritance_ models are together. 

It may also merit some docstrings/comments to clarify the purposes (as coming back a few months later it's a bit _What was that for again?_)